onnx/backend/test/case/node/softmaxcrossentropy.py

```diff
@@ -10,7 +10,7 @@
 from . import expect
 
 
-def softmaxcrossentropy(x, target, weight=None, reduction='mean'):  # type: ignore
+def softmaxcrossentropy(x, target, weight=None, reduction='mean', ignore_index=-100):  # type: ignore
```

Comment:
maybe set ignore_index to None?
(Unlike PyTorch, the spec does not say that the default is -100.)